onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/OperatorUtility.h

```diff
@@ -65,6 +65,7 @@ namespace Dml
 
     // Given an axis in ONNX axis numbering, return the axis adjusted for DML based on how the sizes have been coerced.
     // Note this function presumes the axis attribute is relative to the first input tensor (which is always the case).
+    uint32_t GetDmlAdjustedAxis(int32_t onnxAxis, const MLOperatorKernelCreationContext& kernelCreationContext, uint32_t dmlDimCount, uint32_t firstInputIndex);
     uint32_t GetDmlAdjustedAxis(int32_t onnxAxis, const MLOperatorKernelCreationContext& kernelCreationContext, uint32_t dmlDimCount);
```

Comment:
Should we assign 0 as a default value to firstInputIndex parameter and remove the 2nd overloaded method?